.pep8speaks.yml

```diff
@@ -3,9 +3,18 @@
 scanner:
     diff_only: True  # If True, errors caused by only the patch are shown
 
+# Opened issue in pep8speaks, so we can directly use the config in setup.cfg
+# (and avoid having to duplicate it here):
+# https://github.com/OrkoHunter/pep8speaks/issues/95
+
 pycodestyle:
     max-line-length: 79
-    ignore:  # Errors and warnings to ignore
+    ignore:
+        - W503,  # line break before binary operator
         - E402,  # module level import not at top of file
+        - E722,  # do not use bare except
         - E731,  # do not assign a lambda expression, use a def
-        - W503   # line break before binary operator
+        - E741,  # ambiguous variable name 'l'
```

Comment:
Hey, this seems like a good change to me! Quick question I recently actually enforced E741 in pep8speaks config, #22795. Hence this error will be flagged on any new PRs which is the behaviour we want? I was going to follow up remove any final instances of this in codebase. See #22122

Reply:
I think for now is important to have consistency between the CI and pep8speaks, and among all the different files. I don't have a strong opinion about E741, or the errors discussed in #22122, in general I think the less exceptions we have, the better.